Dockerfile

```
@@ -1,7 +1,20 @@
-FROM        quay.io/prometheus/busybox:latest
-MAINTAINER  Alexey Palazhchenko <alexey.palazhchenko@percona.com>
+FROM golang:1.10
```

```suggestion
FROM golang:1.11
```
We can update it to `1.11`